source/seedsource/management/commands/populate_services.py

```diff
@@ -0,0 +1,122 @@
+import os
+import numpy
+import pyproj
+from clover.geometry.bbox import BBox
+from clover.render.renderers.stretched import StretchedRenderer
+from clover.utilities.color import Color
+from django.db import transaction
+from django.core.management.base import BaseCommand
+from ncdjango.models import Service, Variable
+from netCDF4 import Dataset
+
+# TODO - add 'resume' capability for missing or new services
```

Comment:
It looks like this functionality already exists (existing services are skipped).